tests/conftest.py

```diff
@@ -1,8 +1,73 @@
+from typing import AsyncGenerator
+
 import pytest
+import sqlean
+from phoenix.db import models
+from psycopg import Connection
+from sqlalchemy.ext.asyncio import AsyncSession, create_async_engine
+from sqlalchemy.orm import sessionmaker
 
 
 @pytest.fixture
 def openai_api_key(monkeypatch: pytest.MonkeyPatch) -> str:
     api_key = "sk-0123456789"
     monkeypatch.setenv("OPENAI_API_KEY", api_key)
     return api_key
+
+
+def create_async_postgres_engine(psycopg_connection: Connection) -> sessionmaker:
+    connection = psycopg_connection.cursor().connection
+    user = connection.info.user
+    password = connection.info.password
+    database = connection.info.dbname
+    host = connection.info.host
+    port = connection.info.port
+    async_database_url = f"postgresql+asyncpg://{user}:{password}@{host}:{port}/{database}"
+    return create_async_engine(async_database_url, echo=True)
+
+
+def create_async_sqlite_engine() -> sessionmaker:
+    return create_async_engine("sqlite+aiosqlite:///:memory:", module=sqlean, echo=True)
```

Comment:
Suggestion: make these private variable names and move to bottom if they are local to this file